resources/views/admin/layouts/base.blade.php

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    <head>
```

Comment:
I don't think so, since if Symfony has a template engine which is Twig, so Laravel also has its own template engine which is Blade. Most of the Laravel developers used Blade as the Template engine and for me, it's a little weirdo if we use different template engines.

Reply:
noted no problem